frontend/pages/about.tsx

```diff
@@ -7,11 +7,11 @@ const About: NextPage = () => {
       <Head>
         <title>About | Manipal OSF</title>
       </Head>
-      <div className='m-auto grid gap-10 place-self-center text-center dark:text-white lg:w-[60rem]'>
+      <div className='grid gap-10 m-auto text-center place-self-center text-secondary dark:text-secondary-dark'>
         <h1 className='text-3xl font-bold md:text-5xl lg:text-7xl'>
           Our vision
         </h1>
-        <p className='text-2xl md:text-4xl lg:text-6xl'>
+        <p className='text-2xl md:text-4xl lg:text-6xl text-accent dark:text-accent-dark'>
```

Comment:
I don't think theming the entire paragraph with the accent color is a good idea. Rather, this can be the secondary color and the background of the page can be primary. Same follows for the `syllabus/` route

Reply:
Fixing this

Reply:
I've removed that route completely

Reply:
Replaced it with the routes that we've discussed during the meeting.